device/src/bin/v2.rs

```diff
@@ -532,26 +535,13 @@ where
                     }
                     AnimationProgress::FinalTick => {
                         let ui_event = match prompt {
-                            Prompt::KeyGen {
-                                key_name, key_id, ..
-                            } => UiEvent::KeyGenConfirm {
-                                key_name: key_name.clone(),
-                                key_id: *key_id,
-                            },
+                            Prompt::KeyGen { .. } => UiEvent::KeyGenConfirm,
                             Prompt::Signing(_) => UiEvent::SigningConfirm,
                             Prompt::NewName { new_name, .. } => {
                                 UiEvent::NameConfirm(new_name.clone())
                             }
-                            Prompt::DisplayBackupRequest((_key_name, key_id)) => {
-                                UiEvent::BackupRequestConfirm(*key_id)
-                            }
-                            Prompt::ConfirmFirmwareUpgrade {
-                                firmware_digest,
-                                size,
-                            } => UiEvent::UpgradeConfirm {
-                                firmware_digest: *firmware_digest,
-                                size: *size,
-                            },
+                            Prompt::DisplayBackupRequest { .. } => UiEvent::BackupRequestConfirm,
+                            Prompt::ConfirmFirmwareUpgrade { .. } => UiEvent::UpgradeConfirm,
```

Comment:
looks like I was just reducing stuff sent here since it wasn't actually used for anything.